sdks/python/apache_beam/io/textio.py

```diff
@@ -383,7 +389,7 @@ def open(self, temp_path):
     if self._header is not None:
       file_handle.write(self._header)
       if self._append_trailing_newlines:
-        file_handle.write(b'\n')
+        file_handle.write('\n')
```

Comment:
I think these are required for Python 3.

Reply:
done